### PR TITLE
💄Feature/17/footer

### PR DESCRIPTION
**Pull request by ashby-git**
**What did you do?**
- Created a footer for the bottom of the website.
- It's responsive 
- The top button has functionality

**Screenshot**
14  Footer - reg
14  Footer - hover

**How to Check Feature/Bug Fix**
```
import './App.css';
import Footer from './components/footer/footer.component';

function App() {
  return (
      <Footer />
  );
}

export default App;
```

**Link to the Design**
(https://xd.adobe.com/view/c0f0ee74-781d-434f-9960-3ff85a002849-95a3/specs/?fbclid=IwAR3bcz-BSXJGrgTXDSVBv0y-PYnSKaPM075W3UEhqyWRb_iHMHv7nu77Hq4)

### CODE REVIEW

File: src/components/footer/footer.component.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+import './footer.styles.scss';
+
+const scrollToTop = () => {
+    window.scrollTo(0, 0);
+};
+
+const Footer = () => (
+    <div className="footer-background">
+        <div className="footer-top-section">
```
Comment (DanArcherOnline) [inline src/components/footer/footer.component.jsx:15]: should probably match this with the bottom section by adding or removing `-section`

File: src/components/footer/footer.component.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                    <div className="page-info-group">
+                        <p className="page-name">NEWS</p>
+                        <p className="page-content">新着情報</p>
+                        <p className="page-content">メディア掲載実績</p>
+                    </div>
+
+                    <div className="page-info-group">
+                        <p className="page-name">CONTACT</p>
+                        <p className="page-content">FAQ - よくある質問</p>
+                        <p className="page-content">問い合わせフォーム</p>
```
Comment (DanArcherOnline) [inline src/components/footer/footer.component.jsx:65]: all of these should be links. in another task you'll have them auto scroll to their respective section when clicked.
the design doesn't show how these links look when interacted with so you can get creative.
try adding styles for all the [link states](https://www.bitdegree.org/learn/css-link-color#link-states)

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
tighty whities.
as we discussed before, the "go to top" button is hidden when squishing the browser, but let's fix that after joining all the sections together.
so we don't forget, make a bug issue and post it in this PR!

File: src/components/footer/footer.component.jsx (hunk 1 above)
Comment (ashby-git) [reply src/components/footer/footer.component.jsx]: changed

File: src/components/footer/footer.component.jsx (hunk 2 above)
Comment (ashby-git) [reply src/components/footer/footer.component.jsx:65]: added some styling

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
- [x] turn non header titles in footer into clickable links
- [x] style all 4 link states for non header titles
- [x] style 3 link states for header titles in footer
  - normal, hover, pressed down
- [x] make privacy policy a link to something
- [ ] turn facebook and twitter icons into links

**Pull request merged**